code/controllers/subsystem/vote.dm

```diff
@@ -111,6 +115,32 @@ SUBSYSTEM_DEF(vote)
 
 	current_vote.choices_by_ckey[voter.ckey] = their_vote
 	current_vote.choices[their_vote]++
+
+	return TRUE
+
+/**
+ * Submit a vote using the Approval Voting (AV) system
+ * Any number of selections per person, and the selection with the most votes wins.
+ */
+/datum/controller/subsystem/vote/proc/submit_av_vote(mob/voter, their_vote)
+	if(!current_vote)
+		return
+	if(!voter?.ckey)
+		return
+	if(CONFIG_GET(flag/no_dead_vote) && voter.stat == DEAD && !voter.client?.holder)
```

Comment:
Was this the code that was here in the other votes? The condition here confuses me a little

Reply:
Yeah that's existing code.

Reply:
Alright, sounds fine, I was trying to check in my head if this actually works